### ISSUE TITLE
cargo install cargo-generate fails to build and install, Ubuntu 19

### ISSUE THREAD

**Issue by stevematdavies** [ashleygwilliams/cargo-generate#160]
> 
`cargo install cargo-generate `

build ends in error
```
HOST = x86_64-unknown-linux-gnu
$TARGET = x86_64-unknown-linux-gnu
openssl-sys = 0.9.47

', /.......cargo/registry/src/github.com-1ecc6299db9ec823/openssl-sys-0.9.47/build/find_normal.rs:150:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: failed to compile `cargo-generate v0.3.0`, intermediate artifacts can be found at `/tmp/cargo-install2MOAJQ`

Caused by:
  build failed
```
OS: Ubuntu 19:04
Gnome 3.32.1
Intel® Core™ i7-2720QM
cargo 1.36.0-nightly (6be12653d 2019-04-19)
rustc 1.36.0-nightly (3991285f5 2019-04-25)
rustup 1.18.1 (462683b3a 2019-04-25)

**Comment by k0pernicus** [comment]
I just tried on my GNU/Linux system (the same as yours), and the build was OK (same Nightly version as yours too).

As it seems the build error comes from `openssl-sys`, did you installed the required packages listed here: https://docs.rs/openssl/0.10.23/openssl/?

**Comment by k0pernicus** [comment]
@stevematdavies Is this issue still in pending? :)

**Comment by stevematdavies** [comment]
Suppose not, it's taken so long to get any replies, I gave up

**Comment by k0pernicus** [comment]
> it's taken so long to get any replies
I don't think you received my response notification unfortunately, as I answered it the same day as your question :/

**Comment by ashleygwilliams** [comment]
cc @xortive i think the change you made for the wrangler readme would help here! (https://github.com/cloudflare/wrangler/pull/407)

**Comment by crazcalm** [comment]
I ran into the same issue.

TL;DR: install openssl
```
# On Ubuntu
sudo apt-get install libssl-dev
# On Arch Linux
sudo pacman -S openssl
# On Fedora
sudo dnf install openssl-devel
```

The fix can be found here: https://github.com/sfackler/rust-openssl/issues/1150

**Comment by silvanocostanzo** [comment]
thanks @crazcalm, it worked for me
